pontoon/base/models.py

```diff
@@ -1,11 +1,15 @@
+from __future__ import unicode_literals
 from __future__ import division
+from django.utils.encoding import python_2_unicode_compatible
```

Comment:
This should go with the other ``django`` related imports. Note that this is how we usually sort imports: 

0. ``__future__``
1. standard library
2. dependencies
3. framework
4. local code

And then we sort them by alphabetical order inside each section.